src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -69,9 +72,10 @@ public boolean extractQuestionDetails(
     
     private List<String> getRubricChoices(Map<String, String[]> requestParameters, int numOfRubricChoices) {
         List<String> rubricChoices = new ArrayList<String>();
-        for(int i = 0 ; i<numOfRubricChoices ; i++) {
-            String choice = HttpRequestHelper.getValueFromParamMap(requestParameters, Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_CHOICE + "-" + i);
-            if(choice != null) {
+        for (int i = 0; i < numOfRubricChoices; i++) {
+            String choice = HttpRequestHelper.getValueFromParamMap(requestParameters, 
+                                              Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_CHOICE + "-" + i);
```

Comment:
This indentation is misleading (parameter aligned with method name). Indent a few more spaces.